client/blocks/edit-gravatar/index.jsx

```diff
@@ -156,40 +158,82 @@ export class EditGravatar extends Component {
 		}
 	}
 
+	renderEmailVerificationNotice = () => {
```

Comment:
this can be a normal class method. it'll pick up the `this` binding from the call site.